tic_tac_toe/game.py

```diff
@@ -24,7 +31,16 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+    valid_pos = []
+    for row in range(3) :
```

Comment:
You can declare the valid positions more concisely with `[(row,col) for row in range(3) for col in range(3)]`